From 7d5f436c7d6b12e0345e45e27948f0e10ae12e55 Mon Sep 17 00:00:00 2001
From: Shawn Landden <shawn@churchofgit.com>
Date: Tue, 10 Dec 2013 09:28:26 -0800
Subject: [PATCH] core/manager: remove infinite loop

(cherry picked from commit 145b1f799ff6712f333cef92c44218254a1320c5)
(cherry picked from commit f980d7eccb5a25a174454dd7ba7d4ebc8a9b5a42)
---
 src/core/manager.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/src/core/manager.c b/src/core/manager.c
index 481ec3029e..446c642546 100644
--- a/src/core/manager.c
+++ b/src/core/manager.c
@@ -2184,10 +2184,8 @@ int manager_deserialize(Manager *m, FILE *f, FDSet *fds) {
         }
 
 finish:
-        if (ferror(f)) {
+        if (ferror(f))
                 r = -EIO;
-                goto finish;
-        }
 
         assert(m->n_reloading > 0);
         m->n_reloading --;
